src/scenario.jl

```diff
@@ -29,7 +29,7 @@ function setup_cache(domain::Domain)::NamedTuple
         depth_coeff=zeros(n_sites),  # store for depth coefficient
         site_area=Matrix{Float64}(site_area(domain)'),  # site areas
         TP_data=Matrix{Float64}(domain.TP_data),  # transition probabilities
-        waves=zeros(length(timesteps(domain)), n_species, n_sites)
+        # waves=zeros(length(timesteps(domain)), n_species, n_sites)
```

Comment:
Undo all changes to do with waves. I have temporary changes in PR #381 as we discussed.